Improve grammar, spelling, and punctuation in Usage guide section (#9395)
wagtail/wagtail: docs/topics/writing_templates.md

## Static files (tag)

> Used to load anything from your static files directory. Use of this tag avoids rewriting all static paths if hosting arrangements change, as they might be between development and live environments.

Reviewer: "they might" is referring to the verb "change" in the previous phrase, so adding "be" isn't correct. Rewording for clarity:

```suggestion
Used to load anything from your static files directory. Use of this tag avoids rewriting all static paths if hosting arrangements change, such as when moving from development to a live environment.
```

## Varying output between preview and live

> Sometimes you may wish to vary the template output depending on whether the page is being previewed or viewed live. For example, if you have a visitor tracking code such as Google Analytics in place on your site, it's a good idea to leave this out when previewing, so that editor activity doesn't appear in your analytics reports. Wagtail provides a `request.is_preview` variable to distinguish between preview and live:

Reviewer: "visitor tracking code" = code that tracks visitors. (It's the equivalent of "man-eating fish" versus "man eating fish" :-) )

```suggestion
Sometimes you may wish to vary the template output depending on whether the page is being previewed or viewed live. For example, if you have visitor-tracking code such as Google Analytics in place on your site, it's a good idea to leave this out when previewing, so that editor activity doesn't appear in your analytics reports. Wagtail provides a `request.is_preview` variable to distinguish between preview and live:
```
